doc/Network.mdwn

```diff
@@ -1,37 +1,67 @@
 # Network infrastructure
 
-## VLANs
+## Networks
 
-NOC runs 2 core switches (one in each room), carrying a bunch of VLANs:
-- 33 is the management VLAN (192.168.33.0/24);
-- 127 is the LAN (192.168.127.0/24);
-- 255 (`0xFF`) is our Funkfeuer VLAN.
+NOC operates a number of networks, available as tagged VLANs on the core
+switches (one in each half of the hackerspace).  These networks are:
 
-The switches have hostnames `sw{0,1}.mgmt.realraum.at`, and the WiFi access
-points are similarly `ap{0,1}.mgmt.realraum.at`. `0` denotes the main room, and
-`1` denotes Wöhnung 2.
+| Network    | VLAN id | Extra subnets    |
+|------------|---------|------------------|
+| Management |      32 | --               |
+| IoT        |      33 | --               |
+| Services   |      34 | --               |
+| Public     |      36 | 89.106.211.64/27 |
+| Guests     |     127 | --               |
+| Members    |     128 | 89.106.211.32/27 |
+| `0xFF`     |     255 | --               |
+
+
+### Conventions
+
+We use a number of conventions to make things more consistent:
+
+- The DNS zone for a given network is `NET.realraum.at`, with the exception
```

Comment:
hmmm NET is not really the name as seen above i.e. Managment vlan has hostname `mgmt.realraum.at`
I propose using the "short" name above and add a description column to it so the above table looks like this:

```
+| Name | VLAN id | Extra subnets    | Description                        |
+|------|---------|------------------|------------------------------------|
+| mgmt |    32   | --               | Managment interfaces for routers, switches, etc... |
...
```
The description field will also help to understand the difference between `svc` and `public`.

Reply:
:+1: makes sense.

Reply:
Done  :)